dotnet-desktop-guide/framework/wpf/advanced/how-to-get-print-system-object-properties-without-reflection.md

```diff
@@ -7,6 +7,7 @@ dev_langs:
 helpviewer_keywords: 
   - "PrintSystemObject [WPF], getting properties"
 ms.assetid: 43560f28-183d-41c1-b9d1-de7c2552273e
+description: Learn how the System.Printing.IndexedProperties namespace provides a means to getting this information without using reflection.
```

Comment:
It seems some info is missing. It is not clear what "this information" refers to? Are you able to replace "this information" with a noun?